### PR TITLE
Fix BPEL process Jobs not resumed issue

### PR DESCRIPTION
**Pull request by Drifftr**
## Purpose
[BPS][Intermittent] BPEL process Jobs not resumed after mutiple restart in mesos cluster
https://github.com/wso2/product-ei/issues/2833

### CODE REVIEW

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java [hunk 1]
```diff
@@ -896,6 +905,25 @@ public void onProcessStoreEvent(ProcessStoreEvent processStoreEvent) {
...
 
+    /**
+     * Function to check whether this nos node is manager node by checking registry read/write ability
+     *
+     * @return true if this is manager node, false otherwise
+     */
+    private boolean isManagerNode() {
+        try {
+            //get config registry of super tenant
+            Registry configRegistry = BPELServiceComponent.getRegistryService().getConfigSystemRegistry(-1234);
```
Comment (indikasampath2000) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java]: Better to replace -1234 with MultitenantConstants.SUPER_TENANT_ID

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java [hunk 2]
```diff
@@ -1009,11 +1037,52 @@ public boolean isLeader() {
             HazelcastInstance hazelcastInstance = BPELServiceComponent.getHazelcastInstance();
             Member leader = hazelcastInstance.getCluster().getMembers().iterator().next();
             if (leader.localMember()) {
+                log.debug("ODEClusterImpl#isLeader: true");
```
Comment (indikasampath2000) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java]: Wrap the statement with `if (log.isDebugEnabled())`

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java [hunk 3]
```diff
@@ -1009,11 +1037,52 @@ public boolean isLeader() {
...
                 return true;
             }
+            log.debug("#ODEClusterImpl#isLeader: false");
```
Comment (indikasampath2000) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java]: Wrap the statement with `if (log.isDebugEnabled())`

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java (hunk 1 above)
Comment (Drifftr) [reply components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java]: Fixed in 349c0e4

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java (hunk 2 above)
Comment (Drifftr) [reply components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java]: Fixed in 349c0e4

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java (hunk 3 above)
Comment (Drifftr) [reply components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java]: Fixed in 349c0e4

**Pull request merged**